src/Sarif.Viewer.VisualStudio/Models/SarifErrorListItem.cs

```diff
@@ -617,32 +593,40 @@ internal void RemapFilePath(string originalPath, string remappedPath)
                     }
                 }
             }
+
+            IComponentModel componentModel = (IComponentModel)Package.GetGlobalService(typeof(SComponentModel));
```

Comment:
>IComponentModel [](start = 12, length = 15)

Do we actually need to refresh the tags?

Reply:
Answer is yes, comment why!

---
In reply to: [502079348](https://github.com/microsoft/sarif-visualstudio-extension/pull/214#discussion_r502079348) [](ancestors = 502079348)